jenkins/databricks/run-tests.py

```diff
@@ -18,65 +18,49 @@
 import time
 import os
 import subprocess
+from clusterutils import ClusterUtils
 
-def cluster_state(workspace, clusterid, token):
-  clusterresp = requests.get(workspace + "/api/2.0/clusters/get?cluster_id=%s" % clusterid, headers={'Authorization': 'Bearer %s' % token})
-  clusterjson = clusterresp.text
-  print("cluster response is %s" % clusterjson)
-  jsonout = json.loads(clusterjson)
-  return jsonout
-
-def get_master_addr(jsonout):
-  current_state = jsonout['state']
-  if current_state in ['RUNNING']:
-      driver = jsonout['driver']
-      master_addr = driver["public_dns"]
-      return master_addr
-  else:
-      return None
 
 def main():
   workspace = 'https://dbc-9ff9942e-a9c4.cloud.databricks.com'
   token = ''
-  clusterid = '0617-140138-umiak14'
   private_key_file = "~/.ssh/id_rsa"
-  skip_start = None
   local_script = 'build.sh'
   script_dest = '/home/ubuntu/build.sh'
   source_tgz = 'spark-rapids-ci.tgz'
   tgz_dest = '/home/ubuntu/spark-rapids-ci.tgz'
   ci_rapids_jar = 'rapids-4-spark_2.12-0.1-SNAPSHOT-ci.jar'
-  db_version = '0.1-databricks-SNAPSHOT'
+  # the plugin version to use for the jar we build against databricks
+  db_version = '0.3.0-SNAPSHOT'
   scala_version = '2.12'
   spark_version = '3.0.0'
   cudf_version = '0.16-SNAPSHOT'
   cuda_version = 'cuda10-1'
   ci_cudf_jar = 'cudf-0.14-cuda10-1.jar'
   base_spark_pom_version = '3.0.0'
+  clusterid = ''
 
   try:
-      opts, args = getopt.getopt(sys.argv[1:], 'hs:t:c:p:l:nd:z:j:b:k:a:f:u:m:v:',
-                                 ['workspace=', 'token=', 'clusterid=', 'private=', 'nostart=', 'localscript=', 'dest=', 'sparktgz=', 'cirapidsjar=', 'databricksversion=', 'sparkversion=', 'scalaversion=', 'cudfversion=', 'cudaversion=', 'cicudfjar=', 'basesparkpomversion='])
+      opts, args = getopt.getopt(sys.argv[1:], 'hw:t:c:p:l:d:z:j:b:k:a:f:u:m:v:',
+                                 ['workspace=', 'token=', 'clusterid=', 'private=', 'localscript=', 'dest=', 'sparktgz=', 'cirapidsjar=', 'databricksversion=', 'sparkversion=', 'scalaversion=', 'cudfversion=', 'cudaversion=', 'cicudfjar=', 'basesparkpomversion='])
   except getopt.GetoptError:
       print(
-          'run-tests.py -s <workspace> -t <token> -c <clusterid> -p <privatekeyfile> -n <skipstartingcluster> -l <localscript> -d <scriptdestinatino> -z <sparktgz> -j <cirapidsjar> -b <databricksversion> -k <sparkversion> -a <scalaversion> -f <cudfversion> -u <cudaversion> -m <cicudfjar> -v <basesparkpomversion>')
+          'run-tests.py -s <workspace> -t <token> -c <clusterid> -p <privatekeyfile> -l <localscript> -d <scriptdestinatino> -z <sparktgz> -j <cirapidsjar> -b <databricksversion> -k <sparkversion> -a <scalaversion> -f <cudfversion> -u <cudaversion> -m <cicudfjar> -v <basesparkpomversion>')
```

Comment:
This should go to stderr, but there are a number of other pre-existing prints here that should as well.  Totally OK if this is a low-priority cleanup issue to followup.

Reply:
yeah I didn't change the run-tests.py to go to stderr by default as it wasn't required at this point.  as long as you are ok I'd like to do it in followup when we split that up more